### ISSUE TITLE
[req] torrentland

### ISSUE THREAD

**Issue by vinanrra** [Jackett/Jackett#13768]
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Private

### Tracker details

```markdown
- Name: Torrentland
- URL: https://torrentland.li
- Description: Add HTML Scraping indexer
- Minimum Seeding Time and Minimum Ratio (if Private): 24h seeding time and 0.4 ratio
- Additional Comments: Nothing
```

**Comment by garfield69** [comment]
the indexer for this site already exists as Torrentland (API)
we switched from HTML to API with #13006 as recommended by the unit3d provider via #8692
closing as a duplicate.

**Comment by garfield69** [comment]
is this related to the recent rule change at torrentland that states that accounts that do not have a 1:1 thanks rate will loose their RSS and API access?

**Comment by vinanrra** [comment]
Yes, it's related to that I think we should have both like Pixelados

**Comment by garfield69** [comment]
Yeah, nah. There is little point retaining the API as there is no facility to auto-thank, so API downloads will inevitably lead to the suspension of API access.
I will code a HTML version of the indexer and remove the API indexer.

**Comment by jpalenz77** [comment]
> Yes, it's related to that I think we should have both like Pixelados

I agree with this, I think we should have both, please

**Comment by garfield69** [comment]
@jpalenz77 wrote: I agree with this, I think we should have both, like we used to, please

Yeah, nah. There is little point retaining the API as there is no facility to auto-thank, so API downloads will inevitably lead to the suspension of API access.

**Comment by jpalenz77** [comment]
> @jpalenz77 wrote: I agree with this, I think we should have both, like we used to, please
> 
> Yeah, nah. There is little point retaining the API as there is no facility to auto-thank, so API downloads will inevitably lead to the suspension of API access.

Sorry but I disagree, some of us still get into the tracker to manually thank and fullfill the (silly) rule... and I rather use the API than user/pass

**Comment by garfield69** [comment]
the purpose of jackett is to support the *arr apps to **automate** downloading.
if you have to manually login to the web site and click on the thank you button, then you might as well just click the download button too and leave jackett out of it.
I see no benefit in keeping the api for one user when the html indexer gets the job done for everyone.

**Comment by jpalenz77** [comment]
I do it in batches but yeah, Ok, I get the point,

**Comment by ajimix** [comment]
> The problem with using html is that it triggers vpn warnings if you're using a seedbox. (that never happened with api key).

same issue here, it triggers VPN warning when using seedbox. I would like also to have the API option again that didn't trigger the VPN warning

**Comment by garfield69** [comment]
OK fine. I'll add an API version back, with an info for the config warning about the need to thank via browser.

**Comment by garfield69** [comment]
v0.21.449

**Comment by garfield69** [comment]
https://torrentland.li/ > ERR_CONNECTION_TIMED_OUT

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
My API key is not recognised, and checking my account I find that I am unable to reset token to generate a new one (button diabled). 
Just me?

**Comment by vinanrra** [comment]
@garfield69 @ilike2burnthing they have disabled the API due to a problem they had

**Comment by Alice89634** [comment]
The alternative mode works via login.

Is it possible that they detect this traffic? They say they will ban users who use automations.

**Comment by vinanrra** [comment]
They had a problem not with automations but another problem, and they decided to block automations, i don't know why because they aren't related, there is more info at their Telegram channel, but it isn't related with the traffic or the tool

**Comment by ilike2burnthing** [comment]
Is the API block supposed to be temporary, while they get the other issue fixed?

**Comment by vinanrra** [comment]
We don't know they just said they are blocked for now

**Comment by Alice89634** [comment]
The problem is with a member that take RSS screenshots and report torrentland for DMCA and the admins prefers disable RSS and API for all users and automations is not permited now.
My question is now about if is secure the alternative connection with login

**Comment by vinanrra** [comment]
> The problem is with a member that take RSS screenshots and report torrentland for DMCA and the admins prefers disable RSS and API for all users and automations is not permited now. My question is now about if is secure the alternative connection with login

I think the only way the have to 100% track you it's by user agent, so if you change the user agent you should be "safe"

**Comment by ilike2burnthing** [comment]
The credentials login indexer is working fine and just scrapes the site, but we'd need to get the OK from the site admins to continue using this. However, the description of the issue from @Alice89634 would suggest that they would oppose the credentials login indexer, as this rogue user could just as easily use Jackett.

Problem is though, even if we remove it, the user can just use and old version, that only applies to future releases.

**Comment by garfield69** [comment]
v0.22.1556

**Comment by hectorzin** [comment]
TORRENTLAND:


> We want you to remove the indexers from all future versions for now
> 
> If we ever decide to activate it we will contact you
> 
> Thank you


ironic... sony pictures, paramount pictures, disney, etc... also want they remove all the movies from his tracker!
but jackett cannot have indexers?

A piracy site can have rules now?

I R O N I C
